CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [#5077](https://github.com/bbatsov/rubocop/pull/5077): Add new `Rails/CreateTableWithTimestamps` cop. ([@wata727][])
 * Add new `Style/ColonMethodDefinition` cop. ([@rrosenblum][])
 * Add new `Style/ExtendSelf` cop. ([@drenmi][])
+* Relax version constraint on rainbow, allow 3. ([@jaredbeck][])
```

Comment:
Is it a new feature?

Reply:
Good point. Such changes shouldn't be reflected in the changelog, as they're not user-visible.